### DISCUSSION TITLE
Synology guide does not work.  Needs revision?

### DISCUSSION THREAD

**Discussion by CaligulaCa3sar** [Flexget/Flexget#2822] [Q&A] [answered]
Via Google searches, I can see there were plenty of topics of discussion around this subject on discuss.flexget.com but because of the previous hosting issues, this is now inaccessible.  What a pity for that resource to have apparently gone up in smoke.

First of all, the current versions of DSM and Docker on Synology devices doesn't seem to correctly interpret the --build-arg attribute so I had to hardcode the user ID in the Dockerfile to build the image.

Secondly, now I have an image, I try to deploy a new container from the image and get the following error when running it:

"Could not instantiate manager: No configuration file found."

The container is then immediately stopped.  Please advise.

**Comment by CaligulaCa3sar** [comment, marked as answer]
Seem to have fixed it by manually chown-ing/chmod-ing the mounted config volume to give the assigned user ID +rwx through the DSM CLI.

**Comment by gazpachoking** [comment]
Feel free to update the wiki if there are extra notes that should be in there. 😃
